src/bench/base58.cpp

```diff
@@ -13,7 +13,7 @@
 
 static void Base58Encode(benchmark::State& state)
 {
-    unsigned char buff[32] = {
```

Comment:
Is that because b+32 is passed to EncodeBase58 instead of b+31? So the cleaner way to do it would be to change that function instead of expanding the array by an unneeded element?